js/hw10.js

```diff
@@ -3,62 +3,65 @@
 class List {
   constructor(qttyOfPages) {
     this.qttyOfPages = qttyOfPages;
+    this.form = document.createElement("form");
+    this.select = document.createElement("select");
+    this.input = document.createElement("input");
+    this.div = document.createElement("div");
+    this.div2 = document.createElement("div");
   }
 
   create() {
-    const form = document.createElement("form");
-    const select = document.createElement("select");
-    const input = document.createElement("input");
-    const div = document.createElement("div");
-    const div2 = document.createElement("div");
+    this.input.setAttribute("type", "submit");
+    this.input.setAttribute("value", "GET");
 
-    input.setAttribute("type", "submit");
-    input.setAttribute("value", "GET");
+    document.body.appendChild(this.div);
+    document.body.appendChild(this.div2);
 
-    document.body.appendChild(div);
-    document.body.appendChild(div2);
-
-    form.appendChild(select);
-    form.appendChild(input);
-    div.appendChild(form);
+    this.form.appendChild(this.select);
+    this.form.appendChild(this.input);
+    this.div.appendChild(this.form);
 
     for (let i = 1; i <= this.qttyOfPages; i++) {
```

Comment:
Try to extract to separate class or function

Reply:
Done